src/app/pages/services/components/service-ups/service-ups.component.spec.ts

```diff
@@ -142,4 +143,22 @@ describe('ServiceUpsComponent', () => {
       shutdowntimer: 30,
     } as UpsConfigUpdate]);
   });
+
+  it('allow custom values to be saved as form value for combobox', async () => {
+    const form = await loader.getHarness(IxFormHarness);
+
+    const portSelect = await loader.getHarness(IxComboboxHarness.with({ label: 'Port or Hostname' }));
+
+    await portSelect.writeCustomValue('/my-custom-port');
+
+    const portSelectValue = await portSelect.getValue();
+
+    const saveButton = await loader.getHarness(MatButtonHarness.with({ text: 'Save' }));
+    await saveButton.click();
+
+    const formValue = await form.getValues();
```

Comment:
This still is not very different from what you had before.
Should be something like:
```
// Fill form
// Press save
// expect(ws.call).toHaveBeenCalledWith('ups.update', [expect.objectContaining({
  port: '/my-custom-port',
}]);
```